api/php/DBgateways/RestaurantDbGateway.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+include_once "php/config.php";
+
+class RestaurantDbGateway {
+
+    // private $servername = "localhost";
+    // private $username = "root";
+    // private $password = "foody";
+    // private $db = "foody";
+
+
+    private $dbConnection = null;
+
+    public function __construct() {
+        $this->dbConnection = new DbConnecter();
+    }
+
+    // public function getRestaurants () {
+
+    // }
+
+    public function validateRestaurant (string $restaurantName, string $restaurantAddress, string $restaurantPhone, string $restaurantWebsite) {
+        $restaurantName = $this->dbConnection->cleanInput($restaurantName);
+        $restaurantAddress = $this->dbConnection->cleanInput($restaurantAddress);
+        $restaurantPhone = $this->dbConnection->cleanInput($restaurantPhone);
+        $restaurantWebsite = $this->dbConnection->cleanInput($restaurantWebsite);
+        $restaurantName = $this->dbConnection->cleanInput($restaurantName);
+        $restaurantAddress = $this->dbConnection->cleanInput($restaurantAddress);
```

Comment:
Lines 24-29: What is cleanInput function? I am trying to find the changes in DbConnector file but I do not see it in this PR request. Please let me know!